src/Microsoft.Azure.ServiceBus/Amqp/Framing/AmqpCorrelationFilterCodec.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Amqp.Framing
+{
+    using System.Text;
+    using Azure.Amqp;
+    using Azure.Amqp.Encoding;
+
+    sealed class AmqpCorrelationFilterCodec : AmqpFilterCodec
+    {
+        public static readonly string Name = AmqpConstants.Vendor + ":correlation-filter:list";
+        public const ulong Code = 0x000001370000009;
```

Comment:
This was required to convert if-else to switch as switch needs compile-time constants

---
In reply to: [128847742](https://github.com/Azure/azure-service-bus-dotnet/pull/192#discussion_r128847742) [](ancestors = 128847742)